src/components/input-date-picker/input-date-picker.e2e.ts

```diff
@@ -580,4 +580,39 @@ describe("calcite-input-date-picker", () => {
     expect(changeEvent).toHaveReceivedEventTimes(1);
     expect(await datepickerEl.getProperty("value")).toEqual(["2022-08-15", "2022-08-20"]);
   });
+
+  it("should position on scroll when overlayPositioning is fixed", async () => {
```

Comment:
Nice! Could you try using `commonTests#floatingUIOwner`? It should cover scrolling updates.

Reply:
The test already has a floatingUIOwner test. Maybe it should be updated to test with `overlay-positioning="fixed"` as well?

Reply:
Tried this and it passed so its not catching the issue.

```tsx
 it("owns a floating-ui 2", () =>
    floatingUIOwner(
      `<calcite-input-date-picker overlay-positioning="fixed" value="2022-11-27" min="2022-11-15" max="2024-11-15"></calcite-input-date-picker>`,
      "open",
      { shadowSelector: ".menu-container" }
    ));
```

Reply:
Hmm... I think the helper might need some tweaking. Can you create a follow-up issue for me? Let's proceed with the existing test here.

Reply:
created: https://github.com/Esri/calcite-components/issues/6838